src/main/webapp/script.js

```diff
@@ -0,0 +1,31 @@
+function loadBody() {
+  console.log("Verifying login...");
```

Comment:
Usually console logs are for testingg, probably should remove them now :)

Reply:
oopsies forgot to delete that, thanks!!